src/Pages/Home.js

```diff
@@ -74,6 +76,76 @@ function Home(){
                     <button className="Font_style_2">Resume</button>
                 </div>
             </div> 
+            {/* Project section */}
+            <div className="Project_section">
+                <Card
```

Comment:
Because you’re re-using the same “Card” component multiple times here, I would recommend splitting up this card to its own component. You can pass in props like title, image, etc. down to the card component to render the exact same thing it’s rendering now.
Making your code modular and easy to read is a huge part of writing clean code. Writing everything inside a single file makes your code hard to read/understand.